tier1-icons.js

```diff
@@ -1,803 +0,0 @@
-import '@polymer/polymer/polymer-legacy.js';
```

Comment:
Won't just deleting these files cause build errors for people that were loading them? Like https://github.com/BrightspaceUI/navigation/blob/74425b5736614d97a39e91cfc3322ab3ae4b5df0/d2l-navigation-button-close.js#L4

Reply:
Nvm, you're just emptying these files but they aren't being deleted.

Reply:
The github diff doesn't make that super clear, everything looks the same other than the small message under "Load Diff":
![image](https://user-images.githubusercontent.com/13419300/62554853-e1aa5e00-b83f-11e9-90bc-85accfd2d500.png)
🤷‍♀

Reply:
Haha weird -- yeah just emptying them out.